server.js

```diff
@@ -93,34 +88,32 @@ app.post("/signup", async (req, res) => {
 })
 
 app.post("/login", async (req, res) => {
-    const loginuser = await User.findOne({ username: req.body.username })
     try {
-        if (loginuser) {
+        const loginuser = await User.findOne({ username: req.body.username })
+        if (!loginuser) {
+            res.redirect('/');
+        } else {
```

Comment:
The idea in doing it this was is that you can `return` inside the if, then you don't need the `else`, keeping you code flat